Extension/src/LanguageServer/configurations.ts

```diff
@@ -1840,7 +1840,7 @@ export class CppProperties {
         if (this.prevSquiggleMetrics.get(currentConfiguration.name) === undefined) {
             this.prevSquiggleMetrics.set(currentConfiguration.name, { PathNonExistent: 0, PathNotAFile: 0, PathNotADirectory: 0, CompilerPathMissingQuotes: 0, CompilerModeMismatch: 0 });
         }
-        const newSquiggleMetrics: { [key: string]: number } = { PathNonExistent: 0, PathNotAFile: 0, PathNotADirectory: 0, CompilerPathMissingQuotes: 0, CompilerModeMismatch: 0 };
+        const newSquiggleMetrics: { [key: string]: number } = { PathNonExistent: 0, PathNotAFile: 0, PathNotADirectory: 0, CompilerPathMissingQuotes: 0, CompilerModeMismatch: 0, MultiplePathsNotAllowed:0 };
```

Comment:
If you're adding a new telemetry metric, you need to also send it with the telemetry. Look around line 2158.